frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -233,8 +238,20 @@ export const DetachedTestResultsList: any = ({
   const onPatientSelect = (patient: Patient) => {
     setDebounced("");
     setSelectedPatientId(patient.internalId);
+    setShowSuggestion(false);
   };
 
+  const dropDownRef = useRef(null);
+  const showDropdown = useMemo(() => allowQuery && showSuggestion, [
+    allowQuery,
+    showSuggestion,
+  ]);
+  const hideOnOutsideClick = () => {
+    setShowSuggestion(false);
+  };
```

Comment:
If you wrap this in a `useCallback` hook you should be able to safely add the second argument from `useOutslideClick` to the dependency array.

```javascript
const hideOnOutsideClick = useCallback(() => {
  setShowSuggestion(false);
}, []);
```